src/pycrunch/elements.py

```diff
@@ -282,7 +282,10 @@ def status_401(self, r):
         if r.request.url == login_url:
             raise ValueError("Log in was not successful.")
 
-        creds = {"email": self.session.email, "password": self.session.password}
+        if self.session.token:
+            creds = {"token": self.session.token}
```

Comment:
What is the best way to test this?

I tried:

```
site.session.post("https://alpha.crunch.io/api/public/login/", json={"token": site.session.token}, headers={"Content-Type": "application/json"})
```

but got the following:

```
ClientError: (<Response [400]>, 'https://alpha.crunch.io/api/public/login/', {'status': '400 Bad Request', 'message': 'Login failure during existing session'})
```

Reply:
Done in https://github.com/Crunch-io/pycrunch/pull/78/commits/1890d5e8e891506c53fed1744ac0f2e6b1d14782